examples/async/src/lib.rs

```diff
@@ -38,12 +49,14 @@ fn decode_hex(s: &str) -> Result<Vec<u8>, ParseIntError> {
 }
 
 async fn handle_async() {
-    msg::send(0.into(), b"LOG", 0);
-    let dest = unsafe { PING_PROGRAM_ID };
-    let another_reply = msg_async::send_and_wait_for_reply(dest, b"PING", 50_000_000, 0).await;
-    let another_reply = String::from_utf8(another_reply).expect("Invalid reply: should be utf-8");
-    if another_reply == "PONG" {
-        msg::reply(b"PING", 2_000_000, 0);
+    let reply1 = msg_async::send_and_wait_for_reply(unsafe { DEST_0 }, b"PING", 5_000_000, 0).await;
```

Comment:
can we just sent all `msg::gas_available()` here and below?

Reply:
Tests don't work while sending with `gas_available`. It seems that gas is not sufficient for program execution after sending.

Reply:
Gas_available() minus some constant

Reply:
It doesn't work, will be fixed sepearately.